specification/eventhub/resource-manager/Microsoft.EventHub/preview/2018-01-01-preview/EventHub-preview.json

```diff
@@ -1241,6 +1241,54 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/namespaceOperations/{operationResultId}": {
+      "get": {
+        "tags": [
+          "NamespaceOperations"
+        ],
+        "operationId": "NamespaceOperations_get",
+        "x-ms-examples": {
+          "NamespaceOperationsGet": {
+            "$ref": "./examples/NameSpaces/EHNameSpaceOperationsIDGet.json"
+          }
+        },
+        "description": "Get the status of long running Namespace operation",
+        "externalDocs": {
+          "url": "https://msdn.microsoft.com/en-us/library/azure/mt639408.aspx"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/LocationNameParameter"
+          },
+          {
+            "$ref": "#/parameters/OperationResultIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Namespace long running operation successfully completed"
+          },
+          "202": {
+            "description": "Namespace long running operation successfully accepted."
+          },
+          "204": {
+            "description": "No content"
+          },
```

Comment:
I am not sure about the purpose of this API. Is it something belong to LRO? https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-long-running-operation